Tests/test_peep_pip.py

```diff
@@ -0,0 +1,255 @@
+import unittest
+import math
+
+import numpy as np
+from scipy.interpolate import interp1d
+
+from peep_pip import PEEP, PIP
+from filter_rms_error import filter_rms_error
+from process_sample_data import ProcessSampleData
+
+
+fake_data_safety_factor = 8
+actual_data_safety_factor = 2
+
+
+class TestPEEP(unittest.TestCase):
+
+    def test_sin_input(self):
+        class PEEPTester(PEEP):
+            def __init__(self, dt):
+                super().__init__(dt, maximum_pressure=-0.9)
```

Comment:
negative max pressure?

Reply:
`maximum_pressure` refers to the max pressure that can be PEEP.  Maybe I should rename it to `maximum_peep`.
It's negative because the PEEP for sin[x] is -1 (-0.9 just gives the filter a little room).